interface/new/new_comprehensive.php

```diff
@@ -80,6 +80,9 @@ function getSearchClass($data_type) {
 <head>
 <?php html_header_show(); ?>
 
+<?php
+require_once "{$GLOBALS['srcdir']}/templates/standard_header_template.php";
```

Comment:
this script seems to include fancybox, so need to figure out if it's really used. If so, then will only be able to do "bootstrap light light" here.